src/mpid/ch4/netmod/ofi/ofi_send.h

```diff
@@ -37,6 +37,19 @@
         copy_low_latency - use a low-latency copy engine
         yaksa - use Yaksa
 
+    - name        : MPIR_CVAR_CH4_OFI_EAGER_THRESHOLD
+      category    : CH4_OFI
+      type        : int
+      default     : -1
+      class       : none
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_LOCAL
+      description : >-
+        Messages below MPIR_CVAR_CH4_OFI_EAGER_THRESHOLD will be sent eagerly. Messages
+        above the threshold will perform redezvous handshake before sending the data.
+        If set to -1, the messages will be sent eagerly up to provider max_msg_size.
```

Comment:
I don't want to give the impression that messages up to max_msg_size are sent eagerly. How about:
```
Messages below MPIR_CVAR_CH4_OFI_EAGER_THRESHOLD will be sent eagerly using fi_tagged interfaces. Messages above the threshold will perform an MPICH-level rendezvous handshake before sending the data. If set to -1, MPICH will only perform rendezvous for messages larger than the provider max_msg_size. Note the MPICH eager/rendezvous threshold is independent of any internal libfabric provider threshold.
```

Reply:
That sounds fine.

Reply:
Applied.